packages/wallets/src/components/WalletTourGuide/WalletTourGuide.tsx

```diff
@@ -19,7 +20,8 @@ const WalletTourGuide = () => {
     const [addMoreWalletsTransformValue, setAddMoreWalletsTransformValue] = useState('');
     const { isMobile } = useDevice();
 
-    const { isFetching, isLoading, isSuccess, switchAccount } = useAuthorize();
+    const { isFetching, isLoading, isSuccess } = useAuthorize();
+    const switchAccount = useWalletAccountSwitcher();
```

Comment:
I know it was there already, but just noticed - but why does the WalletTourGuide needs to switch account within useEffect at all? it looks like a recipe for race hazards ;) probably worth having a separate card to investigate/change that @nijil-deriv